.gitignore

```diff
@@ -1,7 +1,8 @@
 **/orxBuild.h
 cache
 code/bin
-code/build
+code/build/*
+!code/build/template

```

Comment:
These additions make it easier to commit template changes without forcing the additions. This took some experimenting for me to get it right!